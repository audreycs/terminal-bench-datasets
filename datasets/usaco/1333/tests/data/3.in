10
906 527
608 244
263 174
839 281
153 492
987 849
701 390
195 518
410 59
48 971
