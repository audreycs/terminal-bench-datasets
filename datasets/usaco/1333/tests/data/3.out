23
42
50
80
19
31
28
32
30
34
